test/SeqCli.EndToEnd/Support/TestConfiguration.cs

```diff
@@ -1,33 +1,35 @@
 using System;
 using System.Collections.Generic;
 using System.IO;
+using System.Threading;
 
 namespace SeqCli.EndToEnd.Support;
 
 public class TestConfiguration(Args args)
 {
-    static int ServerListenPort => 9989;
+    static int _nextServerPort = 9989;
+    readonly int _serverListenPort = Interlocked.Increment(ref _nextServerPort);
```

Comment:
These changes are to support running each test case against a new, unique server instance. Previously, a failure in one test could leave the server in an unsuitable state for later tests, making it difficult to see which test actually failed.